components/DraggableMap.vue

```diff
@@ -116,6 +118,7 @@ export default {
           this.locating = true
           navigator.geolocation.getCurrentPosition((position) => {
             // Show close to where we think they are.
+            this.locating = false;
```

Comment:
What's the reason for this change?  It looks like if the geolocate fails we'll end up with `locating` set true forever.

Reply:
@misterpekert I think this problem still exists in your latest code.  Please re-check, and also in `Postcode`.

Reply:
@misterpekert This is still an issue in the latest code.

Reply:
I'm notsure what you mean, `navigator.geolocation.getCurrentPosition` has two arguments: `successCallback` and `errorCallback`. I've added `this.locating = false` to both of them. But now, when we don't need this flag, it's enough to call the `callback` method. I've updated the code. I'm not sure what the logic was before, but `navigator.geolocation.getCurrentPosition` never throws an error, so I've added the `this.locationFailed = true` part to the `errorCallback`. Let me know if we should remove try/catch from here.